docs/operator-manual/user-management/index.md

```diff
@@ -397,18 +397,15 @@ Add a `rootCA` to your `oidc.config` which contains the PEM encoded root certifi
 
 ### Sensitive Data and SSO Client Secrets
 
-You can use the `argocd-secret` to store any sensitive data. ArgoCD knows to check the keys under `data` in the `argocd-secret` secret for a corresponding key whenever a value in a configmap starts with `$`. This can be used to store things such as your `clientSecret`. 
-* Any values which start with `$` will :
-  - If value is in form of `$<secret>:a.key.in.k8s.secret`, look to a key in K8S `<secret>` of the same name (minus the `$`), and reads it value.
-  - Otherwise, look to a key in `argocd-secret` of the same name (minus the `$`),
-  to obtain the actual value. This allows you to store the `clientSecret` as a kubernetes secret.
-  Kubernetes secrets must be base64 encoded. To base64 encode your secret, you can run
-  `printf RAW_STRING | base64`.
+`argocd-secret` can be used to store sensititve data which can be referenced by ArgoCD. Values starting with `$` in configmaps is interpreted as follows:
```

Comment:
Minor Spelling mistakes: 
sensititve data - >sensitive data
is interpreted -> are interpreted